sat_vapor_pres/include/sat_vapor_pres_k.inc

```diff
@@ -965,41 +872,43 @@
        endif
      endif !(present (mr))
    else
-     mrs = -999.
+     mrs = -999.0_lkind
      if (present (dmrsdT)) then
-       dmrsdT = -999.
+       dmrsdT = -999.0_lkind
      endif
      if (present (esat)) then
-       esat = -999.
+       esat = -999.0_lkind
      endif
    endif
 
 
- end subroutine compute_mrs_k_2d
+ end subroutine COMPUTE_MRS_K_2D_
 
 !#######################################################################
 
- subroutine compute_mrs_k_1d (temp, press, eps, zvir, mrs, nbad,  &
+ subroutine COMPUTE_MRS_K_1D_ (temp, press, eps, zvir, mrs, nbad,  &
                  mr, hc, dmrsdT, esat,es_over_liq, es_over_liq_and_ice)
 
- real, intent(in),  dimension(:)           :: temp, press
- real, intent(in)                          :: eps, zvir
- real, intent(out), dimension(:)           :: mrs
- integer, intent(out)                      :: nbad
- real, intent(in),  dimension(:), optional :: mr
- real, intent(in),                optional :: hc
- real, intent(out), dimension(:), optional :: dmrsdT, esat
+ real(kind=FMS_SVP_KIND_), intent(in),  dimension(:)           :: temp, press !< temperature, pressure
+ real(kind=FMS_SVP_KIND_), intent(in)                          :: eps, zvir !< EPSILO, ZVIR
+ real(kind=FMS_SVP_KIND_), intent(out), dimension(:)           :: mrs !< mixing ratio at relative humidity
+ integer, intent(out)                      :: nbad !< if temperature is out of range
+ real(kind=FMS_SVP_KIND_), intent(in),  dimension(:), optional :: mr !< vapor mixing ratio
+ real(kind=FMS_SVP_KIND_), intent(in),                optional :: hc !< relative humidity
+ real(kind=FMS_SVP_KIND_), intent(out), dimension(:), optional :: dmrsdT, esat !< d(mrs)/dT, saturation vapor pressure
```

Comment:
I think you need to put each variable on a separate line and comment each separately.